src/main/java/qna/domain/Answer.java

```diff
@@ -20,11 +22,13 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "writer_id")
-    private Long writerId;
+    @ManyToOne
+    @JoinColumn(name = "writer_id")
+    private User writer;
 
-    @Column(name = "question_id")
-    private Long questionId;
+    @ManyToOne
+    @JoinColumn(name = "question_id")
```

Comment:
매핑 잘해주셨습니다 👍 

요구사항에서 `DDL`에 대한 내용을 보시면 `FK `이름이 있는데요, `@ForeignKey`를 통해 지정하실 수 있습니다 😄 
참고 해주시면 감사하겠습니다 🙇 

```suggestion
    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```